lib/Target/CBackend/CBackend.cpp

```diff
@@ -626,7 +626,7 @@ raw_ostream &CWriter::printStructDeclaration(raw_ostream &Out,
 raw_ostream &CWriter::printFunctionAttributes(raw_ostream &Out,
                                               AttributeList Attrs) {
   SmallVector<std::string, 5> AttrsToPrint;
-  for (const auto &FnAttr : Attrs.getFnAttributes()) {
```

Comment:
this is because of the rename operation as seen in this commit https://github.com/llvm/llvm-project/commit/10bbf417377437a66e2c1a575c4607c8de4c6dc0